### PR TITLE
2-fix-oembed-url

### PR DESCRIPTION
**Pull request by falkodev**
Please ensure your pull request follows these guidelines:

- [x] Link to the related issue with requirements and/or clearly describe 1) what problem this solves and 2) the requirements to review it against.
- [x] Update the `CHANGELOG.md` file with the added feature or fix. If there is not yet a heading for an upcoming release, add one following the established pattern.
- [x] Keep the pull request minimal! Break large updates into separate pull requests for individual features/fixes whenever possible. Small requests get reviewed more quickly.
- [x] All tests must pass, including the linters. Run `npm run lint` to test code linting independently.

Thanks for contributing!

------ 

https://gitlab-dcadcx.michelin.net/pa/apostrophe-enhancements/-/issues/1077

When using a malicious URL like this one: https://fr-fr-lab-b2c-experience-michelin-rc.dev-dcadcx.michelin.fr/modules/apostrophe-oembed/query?url=notavalidurl the server crashes - it can yield to a denial of service.

### CODE REVIEW

**Review by abea — APPROVED** [review]

**Review by abea — CHANGES_REQUESTED** [review]
@falkodev I'm not sure if you were waiting for me to merge. Feel free to merge it once approved.

At this point we should update it to put the changelog entry as `Unreleased` and with no date until the next release.

**Comment by falkodev** [comment]
> @falkodev I'm not sure if you were waiting for me to merge. Feel free to merge it once approved.
> 
> At this point we should update it to put the changelog entry as `Unreleased` and with no date until the next release.

Done @abea

**Review by abea — APPROVED** [review]

**Pull request merged**
